app/src/main/java/io/appium/uiautomator2/core/AxNodeInfoHelpers.java

```diff
@@ -44,7 +45,7 @@
 /**
  * This class contains static helper methods to work with {@link AccessibilityNodeInfo}
  */
-public class AccessibilityNodeInfoHelpers {
+public class AxNodeInfoHelpers {
```

Comment:
Codacy found an issue: [The utility class name 'AxNodeInfoHelpers' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/dpgraham/appium-uiautomator2-server/pullRequest?prid=6222243)